src/Provider/Here/Tests/HereTest.php

```diff
@@ -88,19 +101,19 @@ public function testReverseWithRealCoordinates()
         $results = $provider->reverseQuery(ReverseQuery::fromCoordinates(48.8632156, 2.3887722));
 
         $this->assertInstanceOf('Geocoder\Model\AddressCollection', $results);
-        $this->assertCount(1, $results);
+        $this->assertCount(5, $results);
```

Comment:
Any idea why this returns 5 results (instead of 1 previously) ?

Reply:
@jbelien because the URL was malformed: radius value in the URL was not appended to the prox parameter using a coma but using an ampersand so it was considered as a new parameter so it messed up the API call and it returned always one result that way (the maxresults parameter was even ignored)